drivers/wifi/siwx91x/siwx91x_wifi.c

```diff
@@ -787,6 +787,120 @@ static int siwx91x_dev_init(const struct device *dev)
 	return 0;
 }
 
+static int convert_z_sl_twt_req_type(enum wifi_twt_setup_cmd z_req_cmd)
+{
+	switch (z_req_cmd) {
+	case WIFI_TWT_SETUP_CMD_REQUEST:
+		return REQUEST_TWT;
+	case WIFI_TWT_SETUP_CMD_SUGGEST:
+		return SUGGEST_TWT;
+	case WIFI_TWT_SETUP_CMD_DEMAND:
+		return DEMAND_TWT;
+	default:
+		return -EINVAL;
+	}
+}
+
+static int siwx91x_set_twt(const struct device *dev, struct wifi_twt_params *params)
+{
+	sl_status_t status = SL_STATUS_OK;
+	sl_wifi_twt_request_t twt_req;
```

Comment:
Fill this struct this 0 during declaration:
```
    sl_wifi_twt_request_t twt_req = { };
```
... so you won't have to patch your code if some new field are introduced.